### ISSUE TITLE
Question about mammal collections and the sex attribute

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2891]
I haven't bulkloaded mammal records in forever - is the sex attribute required for mammal bulkloads?

**Comment by dustymc** [comment]
No.

**Comment by Jegelewicz** [comment]
@campmlc here is our answer.

**Comment by Jegelewicz** [comment]
@dustymc this field appears yellow (required) when entering mammal data - this is confusing if it isn't required....

**Comment by dustymc** [comment]
A bunch of stuff is "required" in the data entry screen because Curators and CMs asked for that. No attribute is actually required.

**Comment by campmlc** [comment]
I agree this is confusing. Can the requirements be tailored by collection?
If MSB Mamma wants a field, it should be yellow. But if UAM does not, not
yellow?

**Comment by dustymc** [comment]
>Can the requirements be tailored by collection?

That's why it's yellow!

**Comment by campmlc** [comment]
So that means it is required for MSB Mamm, yes?

**Comment by dustymc** [comment]
In that form, for people who are using certain browsers and aren't messing with the client-side code....

**Comment by Jegelewicz** [comment]
It shows up as yellow for UTEP:Mamm data entry - it seems like it is by collection code not individual collection.

**Comment by Jegelewicz** [comment]
Data entry fields should only be highlighted in yellow when they are required by the bulkloader. Currently, the attribute type sex is highlighted for mammal collection, but it is not actually required.
